elements/pl-matrix-component-input/pl-matrix-component-input.mustache

```diff
@@ -92,3 +108,13 @@ No symbolic expressions (those that involve fractions, square roots, variables,
 {{#shortformat}}
 {{#relabs}}(rtol={{rtol}}, atol={{atol}}){{/relabs}}{{#sigfig}}({{digits}} significant figures){{/sigfig}}{{#decdig}}({{digits}} digits after decimal){{/decdig}}
 {{/shortformat}}
+
+{{#format_error}}
```

Comment:
Can we also update the input hints for these elements to be the same as the format-error popovers? At least as much as makes sense, in any case.